src/main/play-doc/developer/BundleConfiguration.md

```diff
@@ -47,7 +58,9 @@ name				 | The human readable name of the bundle. This name appears often in ope
 nrOfCpus             | The number of cpus required to run the bundle (can be fractions thereby expressing a portion of CPU). Required.
 protocol			 | Discussed [below](#Endpoints).
 roles                | The types of node in the cluster that this bundle can be deployed to. Defaults to "web".
-services			 | Discussed [below](#Endpoints).
+service-name	     | Discussed [below](#Endpoints).
+acls     			 | Discussed [below](#Endpoints).
+services             | Deprecated in lieu of `acls`. Discussed [below](#Endpoints).
```

Comment:
Are `services` really deprecated? In my understanding, acls is the default. But users can still declare endpoints with a services declaration instead of using acls.

In fact, we are doing that for the `elastic-search` bundle as well: https://github.com/typesafehub/conductr/blob/master/eslite/build.sbt#L12

So I'd change the wording here and make it more explicit that they either declare `services` or `service-name` and `acls`.